spec/jobs/absolute_id_batch_import_job_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe AbsoluteIdBatchImportJob, type: :job do
+  let(:barcode_entries) do
+    [
+      ["ID", "B", "Stamp"],
+      ["8450", "32101080516626", "7/29/15"],
+      ["8451", "32101080516618", "7/29/15"]
+    ]
+  end
+  let(:sequence_entries) do
+    [
+      ["S",
+       "fkBarcodes",
+       "Size",
+       "CollDes",
+       "CollSeq",
+       "Format",
+       "Stamp",
+       "User",
+       "OK",
+       "OKText",
+       "SOrder",
+       "CollectionCode",
+       "Location",
+       "AtType",
+       "Output",
+       "OutputStamp"],
+      ["7374", "8450", "B", "0", "0", "None", "7/29/15", "msiravo", "TRUE", nil, "1", "Pams / NR / Chinese", "ctsn", nil, "TRUE", nil],
+      ["7375", "8451", "B", "0", "0", "None", "7/29/15", "msiravo", "TRUE", nil, "2", "Pams / NR / Chinese", "ctsn", nil, "TRUE", nil]
+    ]
+  end
+  let(:container_profile_id) { '2' }
+  let(:repository_id) { '4' }
+  let(:ead_id) { 'ABID001' }
+  let(:resource_id) { '4188' }
+  let(:source_client) do
+    stub = stub_aspace_client
+    stub = stub_aspace_container_profiles(client: stub)
+    stub = stub_aspace_locations(client: stub)
+    stub = stub_aspace_repositories(client: stub)
+    stub = stub_aspace_container_profile(container_profile_id: container_profile_id, client: stub)
+    stub_aspace_resource(repository_id: repository_id, resource_id: resource_id, ead_id: ead_id, client: stub)
+  end
+
+  before do
+    AbsoluteId.all.map(&:destroy)
```

Comment:
All this cleanup shouldn't be necessary since we're using transactional fixtures, right?